package.json

```
@@ -8,8 +8,8 @@
   "io_session_manager_api": "https://raw.githubusercontent.com/pagopa/io-auth-n-identity-domain/io-session-manager@1.0.0/apps/io-session-manager/api/internal.yaml",
   "io_session_manager_public_api": "https://raw.githubusercontent.com/pagopa/io-auth-n-identity-domain/io-session-manager@1.0.0/apps/io-session-manager/api/public.yaml",
   "api_public_specs": "https://raw.githubusercontent.com/pagopa/io-backend/v14.3.0-RELEASE/api_public.yaml",
-  "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v14.3.0-RELEASE/api_cgn.yaml",
-  "api_cgn_merchants": "https://raw.githubusercontent.com/pagopa/io-backend/v14.3.0-RELEASE/api_cgn_operator_search.yaml",
+  "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend//refs/tags/v16.2.0-RELEASE/api_cgn.yaml",
```

typo here, please edit only the version without adding the refs tags

```suggestion
  "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v16.2.0-RELEASE/api_cgn.yaml",
```